notebooks/ch17/classification_multiclass.md

```diff
@@ -281,32 +290,40 @@ lr.fit(X_train, y_train)
 
 
 
-We predict on the test data, and use a confusion matrix to evaluate the results:
+We predict on the test data, and use a confusion matrix to evaluate the results.
 
 
 ```python
-# to-do: add x-axis, y-axis labels to confusion matrix
 y_pred = lr.predict(X_test)
-confusion_matrix(y_test, y_pred)
+plot_confusion_matrix(y_test, y_pred)
```

Comment:
I'm not a huge fan of how the diagonal has different shades even though all observations on the diagonal are correctly classified. Do you think this is okay as is, should we normalize (and then lose the annotations about number of observations), or should we manually create a heatmap in `matplotlib` with the diagonals set to black?